packages/itinerary-body/src/stories/OtpUiItineraryBody.story.tsx

```diff
@@ -119,95 +136,126 @@ export const EScooterRentalItinerary = (): ReactElement => (
-  <ItineraryBodyDefaultsWrapper itinerary={parkAndRideItinerary} />
+  <ItineraryBodyDefaultsWrapper
+    itinerary={parkAndRideItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
+  />
 );
 
 export const BikeRentalTransitItinerary = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     itinerary={bikeRentalTransitBikeRentalItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
   />
 );
 
 export const EScooterRentalTransitItinerary = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     itinerary={eScooterRentalTransiteScooterRentalItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
   />
 );
 
 export const TncTransitItinerary = (): ReactElement => (
-  <ItineraryBodyDefaultsWrapper itinerary={tncTransitTncItinerary} />
+  <ItineraryBodyDefaultsWrapper
+    itinerary={tncTransitTncItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
+  />
 );
 
 export const OTP2ScooterItinerary = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper itinerary={otp2ScooterItinerary} />
 );
 
 export const OTP2FlexItinerary = (): ReactElement => (
-  <ItineraryBodyDefaultsWrapper itinerary={flexItinerary} />
+  <ItineraryBodyDefaultsWrapper
+    itinerary={flexItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
+  />
 );
 
 export const IndividualLegFareComponents = (): ReactElement => (
-  <ItineraryBodyDefaultsWrapper itinerary={fareProductsItinerary} />
+  <ItineraryBodyDefaultsWrapper
+    itinerary={fareProductsItinerary}
+    showRouteFares
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
+  />
 );
 
 export const CustomAlertIconsItinerary = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     itinerary={walkTransitWalkItinerary}
     AlertToggleIcon={styled(Bolt).attrs({ size: 15 })``}
     AlertBodyIcon={styled(Bomb).attrs({ size: 18 })``}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
   />
 );
 
 export const ThreeAlertsAlwaysCollapsing = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     alwaysCollapseAlerts
     itinerary={walkTransitWalkItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
   />
 );
 
 export const TwoAlertsAlwaysCollapsing = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     alwaysCollapseAlerts
     itinerary={parkAndRideItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
   />
 );
 
 export const ZeroAlertsAlwaysCollapsing = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     alwaysCollapseAlerts
     itinerary={walkInterlinedTransitItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
   />
 );
 
 export const ThreeAlertsNotAlwaysCollapsing = (): ReactElement => (
   <ItineraryBodyDefaultsWrapper
     alwaysCollapseAlerts={false}
     itinerary={walkTransitWalkItinerary}
+    RouteDescriptionFooter={RouteDescriptionFooterWithWaitTimes}
```

Comment:
Can we remove this prop from a few of these stories? Just to test the defualt behavior. I think the footer only needs to be active for one or two

Reply:
Done.